Autopilot/TelemetryManager/telemetryManager.hpp

```diff
@@ -0,0 +1,17 @@
+#pragma once
+#include "telemetryStateManager.hpp"
+
+class telemetryState;
+
+class telemetryManager
+{
+    public:
+        telemetryManager();
+        inline telemetryState* getCurrentState() const{return currentState;}
+        void execute();
+        void setState(telemetryState& newState);
+        bool dataValid;
+        bool dataError;
+    private:
+        telemetryState* currentState;
```

Comment:
Once my PR gets merged, we will need to create an enum that will store the status of the state machine (ex. COMPLETED_CYCLE, IN_CYCLE, FAILURE_MODE).